homeassistant/components/bittrex/strings.json

```diff
@@ -15,12 +15,20 @@
                 "data": {
                     "markets": "markets"
                 }
+            },
+            "balances": {
+                "title": "Select the cryptocurrency balances",
+                "description": "Select the balances that you would like to track.",
```

Comment:
Suggestion: make it clear that by entering nothing into this form you will get the market-only sensors.

Reply:
Also this description string doesn't appear in en.json which I think indicates you need to return the translation develop command.

Reply:
Thanks, I've added the 'Optional' prefix to the description and synced `en.json` with `strings.json`.